### PR TITLE
LPS-195455 Return parameterEndpoint parameter in MVCResourceCommand

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://liferay.atlassian.net/browse/LPS-195455

@carolmariaabb could you review only this commit https://github.com/liferay-objects/liferay-portal/pull/2541/commits/4b4174a2ff070f7077d221aa0fb5d6492f1326eb, please?

### CODE REVIEW

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectRelationshipInfoMVCResourceCommand.java [hunk 1]
```diff
@@ -54,6 +54,10 @@ protected void doServeResource(
 				"objectRelationshipTypes",
 				ObjectRelationshipUtil.getObjectRelationshipTypes(
 					objectDefinition, _systemObjectDefinitionManagerRegistry)
+			).put(
+				"parameterEndpoint",
```
Comment (carolmariaabb) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectRelationshipInfoMVCResourceCommand.java:58]: @murilostodolni can we rename it to `restContextPath`?

Comment (murilostodolni) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectRelationshipInfoMVCResourceCommand.java:58]: Yes, I agree! But in` edit_object_relationship.jsp` we call `parameterEndpoint`. Do we refactor there as well?

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
I will rebase with master and rerun the CI again. I found the  result of relevant weird.

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:object

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:forward:force

**Pull request closed without merging**